dags/monitor_data_hub_pipeline_health.py

```diff
@@ -0,0 +1,122 @@
+# Note: DagBag.process_file skips files without "airflow" or "DAG" in them
+
+import logging
+
+from datetime import timedelta
+
+from data_pipeline.utils.pipeline_file_io import (
+    get_yaml_file_as_dict
+)
+
+from data_pipeline.utils.pipeline_config import get_env_var_or_use_default
+
+from data_pipeline.utils.dags.data_pipeline_dag_utils import (
+    create_dag,
+    create_python_task
+)
+
+from data_pipeline.monitoring.monitoring_config import (
+    MonitoringConfig
+)
+
+from data_pipeline.monitoring.ping_healthchecks import main as ping
+
+from data_pipeline.monitoring.data_hub_pipeline_health_check import (
+    run_data_hub_pipeline_health_check
+)
+
+LOGGER = logging.getLogger(__name__)
+
+DEPLOYMENT_ENV_ENV_NAME = "DEPLOYMENT_ENV"
+DEFAULT_DEPLOYMENT_ENV = "ci"
+# Do not forgot to add formula:
+MONITORING_CONFIG_FILE_PATH_ENV_NAME = "MONITORING_CONFIG_FILE_PATH"
+
+DAG_ID = "Monitor_Data_Hub_Pipeline_Health"
+# don't forget to add the env var to the formula - 3 hours!
+MONITOR_DATA_HUB_PIPELINE_HEALTH_SCHEDULE_INTERVAL_ENV_NAME = (
+    "MONITOR_DATA_HUB_PIPELINE_HEALTH_SCHEDULE_INTERVAL"
+)
+
+MONITOR_DATA_HUB_PIPELINE_HEALTH_DAG = create_dag(
+    dag_id=DAG_ID,
+    # schedule_interval=os.getenv(
+    #     MONITOR_DATA_HUB_PIPELINE_HEALTH_SCHEDULE_INTERVAL_ENV_NAME
+    # ),
+    dagrun_timeout=timedelta(hours=6)
+)
+
+
+def data_config_from_xcom(context):
+    dag_context = context["ti"]
+    data_config_dict = dag_context.xcom_pull(
+        key="data_config_dict", task_ids="get_data_config"
+    )
+    LOGGER.info('data_config_dict: %s', data_config_dict)
+    deployment_env = get_env_var_or_use_default(
+        DEPLOYMENT_ENV_ENV_NAME, DEFAULT_DEPLOYMENT_ENV)
+    data_config = MonitoringConfig(
+        data_config_dict, deployment_env)
+    LOGGER.info('data_config: %r', data_config)
```

Comment:
just mentioning that this won't log anything useful apart from that `data_config` is not None (or what type it is). That is because `MonitoringConfig` doesn't implement `__str__` or `__repr__`

Reply:
Himm I can add this functions but when you come back we need to talk about it :D as it is not clear for me.